listenbrainz/tests/integration/test_playlist_api.py

```diff
@@ -473,6 +586,86 @@ def test_playlist_private_access(self):
         )
         self.assert404(response)
 
+    def test_playlist_get_playlists(self):
+        """ Test for checking that unauthorized access to private playlists return 404 """
+
+        # create a public and private playlist, then try various forms of access
+        playlist = get_test_data()
+        response = self.client.post(
+            url_for("playlist_api_v1.create_playlist", public="false"),
+            data=ujson.dumps(playlist),
```

Comment:
is this the same as `json=playlist`? Maybe we should standardise on this so that we don't need the content_type flag.